metricbeat/module/prometheus/collector/_meta/data.json

```diff
@@ -13,11 +13,13 @@
     },
     "prometheus": {
         "collector": {
-            "label": {
+            "labels": {
```

Comment:
I would prefer to have here `label` even though I'm aware this is not consitent with most of the other label fields with have. They should also be singular.